pinot-common/src/main/java/org/apache/pinot/common/metrics/BrokerMeter.java

```diff
@@ -50,6 +50,18 @@ public class BrokerMeter implements AbstractMetrics.Meter {
    * That means it can be used to know how many single-stage queries have been started in total.
    */
   public static final BrokerMeter QUERIES_GLOBAL = create("QUERIES_GLOBAL", "queries", true);
+  /**
+   * Number of queries executed, this metric is not global and is attached to a particular replica group.
+   * <p>
+   * At this moment, this counter include single stage queries only.
+   */
+  public static final BrokerMeter REPLICA_QUERIES = create("REPLICA_QUERIES", "routing", false);
+  /**
+   * Number of segment queries executed, this metric is not global and is attached to a particular replica group.
+   * <p>
+   * At this moment, this counter include single stage queries only.
+   */
+  public static final BrokerMeter REPLICA_SEG_QUERIES = create("REPLICA_SEG_QUERIES", "routing", false);
```

Comment:
What are segment queries? If this is not that important we can skip it to be minimal. I suppose the other metric should suffice

Reply:
need an approach to show the blast radius during rollout. The other metric shows the traffic are equal among replicas when we do the isolation group based version upgrade. This metrics aims to count the number of segments hit per replica, we will see most of segments are limited to preferred replica group.

Reply:
I have used hashmap to count the segment selection distribution over replica. The performance regression shouldn't be a concern.